LabExT/Movement/PathPlanning.py

```diff
@@ -464,23 +516,21 @@ def _get_grid_properties(
 
         return grid_size, outline
 
-    def _last_moves_equal(self, last_moves: list, new_move: dict) -> bool:
+    def _last_waypoints_equal(self, next_command: WaypointCommand) -> bool:
         """
         Returns True if all last moves are equal to the new move.
 
         Parameters
         ----------
-        last_moves: list
-            List of last moves
-        new_moves: dict
-            Dict of the new move
+        next_command: WaypointCommand
+            next waypoint commands
         """
-        if len(last_moves) == 0:
+        if len(self.last_moves) == 0:
             return False
 
-        for last_move in last_moves:
+        for last_move in self.last_moves[-self.abort_local_minimum:]:
             for calibration, waypoint in last_move.items():
-                if new_move[calibration] != waypoint:
+                if next_command[calibration].coordinate != waypoint.coordinate:
```

Comment:
@marcoep We found a bug during testing. I forgot to compare the coordinates and not the waypoint. I have updated the routine.